src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/food/Juice.java

```diff
@@ -68,6 +68,10 @@ public ItemConsumptionHandler getItemHandler() {
                     p.addPotionEffect(effect);
                     break;
                 }
+                else if (effect.getType().equals(PotionEffectType.ABSORPTION)) {
```

Comment:
pretty sure this doesnt fit the coding format.
also u can combine them since the code in the block is the same.